src/chainstate/stacks/miner.rs

```diff
@@ -10146,6 +10148,357 @@ pub mod test {
         }
     }
 
+    #[test]
+    fn test_get_block_info_v210() {
```

Comment:
Between this PR and #3144, there's almost 2,000 new lines of tests in this file, and these tests are really only sort of related to the miner module. Only about 2,000 lines of this 10k+ line file are implementation. Both of these things strongly suggest to me that the tests components of this file should be split out (i.e., the fact that tests are being added to this file without any changes otherwise in this module, and the fact that this file is now _huge_ and almost entirely composed of tests). It doesn't have to happen in this PR, but I definitely believe it needs to happen.

Reply:
Sure, I'll open an issue.